charts/azure-js/templates/service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.jsServiceName }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    name: {{.Release.Name}}-vote-js
```

Comment:
Release name should suffice. Drop -vote-js

Reply:
done